src/Tempest/View/composer.json

```diff
@@ -7,11 +7,12 @@
         "masterminds/html5": "^2.9",
         "tempest/core": "self.version",
         "tempest/container": "self.version",
-        "tempest/validation": "self.version"
+        "tempest/validation": "self.version",
+        "tempest/routing": "self.version"
```

Comment:
Why does view need a dependency on routing?

Reply:
My bad, I'll remove it